mysql-test/suite/perfschema/r/show_sanity.result

```diff
@@ -412,6 +412,7 @@ SHOW_MODE	SOURCE	VARIABLE_NAME
 5.6	I_S.SESSION_VARIABLES	INNODB_COMPRESSED_COLUMNS_THRESHOLD
 5.6	I_S.SESSION_VARIABLES	INNODB_COMPRESSED_COLUMNS_ZIP_LEVEL
 5.6	I_S.SESSION_VARIABLES	INNODB_DEADLOCK_DETECT
+5.6	I_S.SESSION_VARIABLES	INNODB_ENCRYPT_TABLES
```

Comment:
The proper way to fix this is to extend `show_sanity.test` itself with a new known global variable rather than re-recording `.result` file.